Automattic/wp-calypso
# Social Login: Offer to create a new account if it doesn't exist

#### Proposed Changes

Add the URL to the signup page in case the user faces the `unknown_user` message when trying to social login:

<img width="925" alt="image" src="https://user-images.githubusercontent.com/36432/182399750-df31b947-273e-40f1-afda-fcca1a992e01.png">

~~I tried passing the service as a query parameter to the signup URL and opening the pop-up on page load, but the browser blocks it. At least Brave did... Chrome also did but it's possible to allow it. Regardless, I'm committing the changes because the user might allow pop-ups by default.~~ Decided to remove the auto-popup behavior.

#### Testing Instructions

See "Testing Instructions" in https://github.com/Automattic/wp-calypso/pull/64957 to set up a working wpcalypso.wordpress.com environment on your local machine.

Try to social login using an account that is not registered in WPCOM and you should see the above message with a link to the signup page. It should work with both Google and Apple.

Clicking on the link should dispatch the `calypso_login_sign_up_link_click` Tracks event.

Fixes https://github.com/Automattic/wp-calypso/issues/65651
